shared/config/config_overlay.go

```
@@ -9,31 +9,211 @@ import (
+	case "", "openai":
+		return true
+	default:
+		return false
+	}
+}
+
+func hasModelCapabilitiesOverride(override ModelCapabilitiesOverride) bool {
+	return override.SupportsReasoningEffort || override.SupportsVisionInputs
+}
+
+func shouldInheritReviewerModelCapabilities(override ModelCapabilitiesOverride, sources map[string]string) bool {
```

golangci-lint (2.12.1)

[error] func shouldInheritReviewerModelCapabilities is unused

(unused)